lib/measured/rails/validations.rb

```diff
@@ -9,9 +9,11 @@ class MeasuredValidator < ActiveModel::EachValidator
 
   def validate_each(record, attribute, measurable)
     measured_config = record.class.measured_fields[attribute]
+    unit_column_name = measured_config[:unit_column_name] || "#{ attribute }_unit"
 
     measured_class = measured_config[:class]
-    measurable_unit = record.public_send("#{ attribute }_unit")
+
+    measurable_unit = record.public_send("#{ unit_column_name }")
     measurable_value = record.public_send("#{ attribute }_value")
```

Comment:
Again, it'd be great to see these consistent. You can pull these out so that you just `public_send(value_field_name)` and `public_send(value_unit_name)`